src/lib/ext-activitylog.js

```diff
@@ -183,6 +184,34 @@ class View {
     this.apiNameFilter.updateFilterCheckboxes(filteredLogs);
   }
 
+  setExistedFilters(updateFilter) {
```

Comment:
I meant to comment about this before; You probably meant "set existing filters" instead of "set existed filters".

`setInitialFilter` may be a better name though, because it does seem that the method can be called only once (as the implementations are not yet prepared to deal with setting the filter after the initialization).

Reply:
Thanks. Updated the name as `setInitialFilter` :-)